### PR TITLE
media-gfx/alembic: bump to version 1.7.11

### PR DESCRIPTION
**Pull request by waebbl**
Closes: https://bugs.gentoo.org/688114
Package-Manager: Portage-2.3.67, Repoman-2.3.14
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

File: media-gfx/alembic/alembic-1.7.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	python? ( boost )
+"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=media-libs/openexr-2.3.0:=
+	boost? ( >=dev-libs/boost-1.65.0:=[python,${PYTHON_USEDEP}] )
+	hdf5? ( >=sci-libs/hdf5-1.10.2:=[zlib(+)] )
+	python? ( >=dev-python/pyilmbase-2.3.0[${PYTHON_USEDEP}] )
+	zlib? ( >=sys-libs/zlib-1.2.11-r1:= )
```
Comment (a17r) [inline media-gfx/alembic/alembic-1.7.11.ebuild]: that slot op is not necessary, introducing subslot on zlib was a mistake.

Comment (waebbl) [reply media-gfx/alembic/alembic-1.7.11.ebuild]: Zlib does have subslots, so why don't we need a subslot operator in this case?

**Comment by waebbl** [comment]
Not really supporting python3. The issue here is `pyilmbase` not supporting python3. I can compile `pyilmbase` against `python:3.6` but it doesn't load and gives an `PyInit_<module>` not defined error, which causes `alembic` to error out during configuration phase.

But there's a lot activity going on, on the openexr repository. It looks like ASWF and the Linux Foundation have taken care of the project, and I expect a new release and python-3 support later this year.

Both projects have several open issues regarding python-3, and also the VFX-2020 reference platform will need it and they encourage software vendors the provide a python-3 tech preview within this year.

File: media-gfx/alembic/alembic-1.7.11.ebuild (hunk 1 above)
Comment (a17r) [reply media-gfx/alembic/alembic-1.7.11.ebuild]: package having subslot does not always automatically mean we need to add a slot operator on all revdeps (see poppler, see Qt packages). in this case, as said, it was simply a mistake introducing it.

Comment (waebbl) [reply media-gfx/alembic/alembic-1.7.11.ebuild]: Ok, thanks for explanation :)

**Pull request closed without merging**